tests/testthat/test-github_document.R

```diff
@@ -0,0 +1,26 @@
+
+test_that("toc has correct identifier", {
+  local_edition(3)
+  skip_on_cran() # avoid pandoc issue on CRAN
+  tmp_file <- withr::local_tempfile()
+  xfun::write_utf8(
```

Comment:
If you use this pattern a lot, you might consider writing something like this:

```R
local_rmd <- function(..., .env = parent.frame()) {
  path <- withr::local_tempfile(envir = .env)
  xfun::write_utf8(c(...), path)
  path
}
```

Reply:
Is `tests/testthat/helpers.R` still a good place ? 
I see there is a new recommandation to use `R/test-helpers.R`.

Reply:
It doesn't really matter, but I now prefer somewhere inside `R/`

Reply:
Ok. I put it in the former as I was used to that. Thanks